conformance/utils/echo/pod.go

```diff
@@ -105,6 +106,9 @@ func makeRequest(t *testing.T, exp *http.ExpectedResponse) []string {
 	}
 
 	host := http.CalculateHost(t, r.Host, protocol)
+	if *flags.OverrideHostHeader != "" {
+		host = *flags.OverrideHostHeader
```

Comment:
looks like this overrides the entire host and not host header only?

Reply:
updated to OverrideHost